components/main/blocks/DKMembershipWidget/DKMembershipWidget.tsx

```diff
@@ -40,6 +40,7 @@ export const DKMembershipWidget: React.FC<{ config?: ConfigurationType }> = ({ c
     field_required_message: "This field is required.",
     submitting_message: "Submitting...",
     membership_fee_text: "Become a member for 50 DKK",
+    failed_submission_message: "Error submitting membership request",
```

Comment:
very minor: maybe "membership form" will be clearer for users

Reply:
Just to make sure, this is what you meant, right?
```suggestion
    failed_submission_message: "Error submitting membership form",
```

Reply:
Yes!